packages/salesforcedx-apex-debugger/src/adapter/apexDebug.ts

```diff
@@ -1034,6 +1039,23 @@ export class ApexDebug extends LoggingDebugSession {
           typerefs: Array.from(exceptionBreakpoints.keys())
         };
         break;
+      case TERMINATE_SESSION_REQUEST:
+        this.sendEvent(new TerminatedEvent());
+        break;
+      case IDLE_SESSION_REQUEST:
```

Comment:
A request from the extension to the adapter to terminate the session because the user took too long to respond to the prompt.

Reply:
@DatGuyJonathan Why is this distinction necessary?